virttest/libvirt_xml/devices/filesystem.py

```diff
@@ -77,6 +78,7 @@ class Binary(base.base.LibvirtXMLBase):
             "lock_posix",
             "flock",
             "thread_pool_size",
+            "open_files",
```

Comment:
Is it better to rename it to 'openfiles_max'? The naming format needs to be consistent with other variables I think. What do you think?

Reply:
@Yingshun Updated. Thanks.